File: project/server/db/models/administradors.js
import DataType from 'sequelize';
import sequelize from '../conf.js';

const Administrador = sequelize.define('administrador', {
    nom: {
        type: DataType.STRING,
        allowNull: false
    },
    cognom: {
        type: DataType.STRING,
        allowNull: false
    },
    usuari: {
        type: DataType.STRING,
        allowNull: false
    },
    password: {
        type: DataType.STRING,
        allowNull: false
    },
    id_torneig: {
        type: DataType.INTEGER,
        allowNull: false,
        references: {
            model: 'Torneig',
            key: 'id_torneig'
        }
    },
    id_personal: {
        type: DataType.INTEGER,
        allowNull: false,
        references: {
            model: 'Personal',
            key: 'id_personal'
        }
    }
});

module.exports = Administrador;

File: project/server/models/administradors.py
from app import db

class Administradors(db.Model):
    id_administrador = db.Column(db.Integer, primary_key=True, autoincrement=True)
    nom = db.Column(db.String(50))
    cognom = db.Column(db.String(50))
    email = db.Column(db.String(150))
    password = db.Column(db.String(250))

    id_personal = db.Column(db.Integer, db.ForeignKey('personal.id_personal'))
    id_torneig = db.Column(db.Integer, db.ForeignKey('torneig.id_torneig'))

    def __repr__(self):
        return '<Administradors: %r' % self.nom